gcloud/pubsub/_gax.py

```diff
@@ -162,17 +162,23 @@ def topic_publish(self, topic_path, messages):
         :raises: :exc:`gcloud.exceptions.NotFound` if the topic does not
                     exist
         """
-        options = CallOptions(is_bundling=False)
+        options = CallOptions(is_bundling=True)
```

Comment:
We can just drop creating `options` altogether (and passing it below, and testing it...)